lib/ddtrace/contrib/resque/resque_job.rb

```diff
@@ -16,26 +16,18 @@ def around_perform(*args)
             yield
             span.service = pin.service
           end
+        ensure
+          pin.tracer.shutdown! if pin && pin.tracer
         end
       end
     end
   end
 end
 
-Resque.before_first_fork do
```

Comment:
are we sure of that change? I remember we added it to handle some cases. We need to deeply test those changes.

Reply:
We want to ship it first as a beta release and before that do another round of testing.

I've tested this change locally when developing it and coupled with some other improvements in separate PRs, it did flush and shutdown as quickly as `SyncWriter`.

This change is mostly driven by the fact that  `SyncWriter` performance deteriorates quite a bit under multithreaded setup.